code/protected/models/Retailer.php

```diff
@@ -457,7 +457,7 @@ public static function getRetailerLog($retailerId){
 
     public function getSalesEmployee(){
         $connection = Yii::app()->db;
-        $sql = 'select ge.id, ge.name from groots_employee as ge left join groots_departments gd on ge.department_id = gd.id where gd.name = "Sales" and ge.status = 1 order by ge.id';
+        $sql = 'select ge.id, ge.name from groots_employee as ge left join employee_department as ed on ed.employee_id = ge.id left join (select id from groots_departments where name = "Sales" limit 1) as gd on gd.id = ed.department_id where ge.status = 1';
```

Comment:
change the query: remove subquery, add join to whole groots_dept table and add condition in where clause